kafka/tests/common.py

```diff
@@ -10,6 +11,7 @@
 HERE = get_here()
 HOST = get_docker_hostname()
 HOST_IP = socket.gethostbyname(HOST)
+COMPOSE_FILE = os.environ["DOCKER_COMPOSE_FILE"]
```

Comment:
This approach is fine, though you may also consider computing it out of the `KAFKA_VERSION` instead to avoid adding an extra env variable

Reply:
that would be neat, thanks! Will do

Reply:
I'm an idiot, the new image have both versions so we just need to use the other image 🙈